checkov/secrets/runner.py

```diff
@@ -103,13 +109,31 @@ def run(self, root_folder, external_checks_dir=None, files=None, runner_filter=R
                         if file not in PROHIBITED_FILES and f".{file.split('.')[-1]}" in SUPPORTED_FILE_EXTENSIONS:
                             files_to_scan.append(os.path.join(root, file))
             logging.info(f'Secrets scanning will scan {len(files_to_scan)} files')
-            for file in files_to_scan:
-                logging.info(f'Scanning file {file} for secrets')
+
+            # TODO: re-enable filter when re-adding `SecretKeyword` plugin
+            scan.get_settings().disable_filters(*['detect_secrets.filters.heuristic.is_indirect_reference'])
+
+            def _scan_file(file_path: str):
+                start = time.time()
                 try:
-                    secrets.scan_file(file)
-                except Exception as e:
-                    logging.warning(f"Secret scanning:could not process file {file}, {e}")
-                    continue
+                    secrets.scan_file(file_path)
+                except Exception as err:
+                    logging.warning(f"Secret scanning:could not process file {file_path}, {err}")
+                end = time.time()
+                scan_time = end - start
+                if scan_time > 10:
+                    logging.info(f'Scanned {file_path}, took {scan_time} seconds')
+
+            with concurrent.futures.ThreadPoolExecutor(max_workers=os.cpu_count()) as executor:
```

Comment:
@nimrodkor Why not use `checkov/terraform/checks/utils/utils.py/run_function_multithreaded`?